### ISSUE TITLE
Work directly with JS objects in `MpClientPlugin::Send`

### ISSUE THREAD

**Issue by nic11** [skyrim-multiplayer/skymp#175]
After #127 is done, it won't make sense to serialize using `JSON.stringify` in TS code anymore

**Comment by Pospelove** [comment]
@nic11 Hey! Could you please remind me where do we call it?

**Comment by nic11** [comment]
@Pospelove Here:
https://github.com/skyrim-multiplayer/skymp/blob/56cca701e977c2d551d3b6eea0de98af661b13ed/skymp5-client/src/front/networking.ts#L52

Also, as I can see, we pass JSON as a on-incoming-packet callback, but it's a separate task (and not so important, I guess)

**Comment by Pospelove** [comment]
Well. Good idea but we do not plan to fix it right? At least without data from profiler
